File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.py
def solution(t, p):
    li = []
    cnt = 0
    
    #p의 길이로 부분문자열
    for i in range(len(t)-len(p)+1):
        li.append(t[i:i+len(p)])
        if int(li[i])<=int(p):
            cnt += 1
    
    return cnt

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.js
function solution(t, p) {
    var answer = 0;
    
    for(let i=0; i<t.length-p.length+1; i++){
        let x = t.slice(i, i+p.length);
        if(x <= +p) answer ++;
    }
    
    return answer;
}